src/main/java/datastax/astra/migrate/AbstractJobSession.java

```diff
@@ -70,6 +70,12 @@ protected AbstractJobSession(CqlSession sourceSession, CqlSession astraSession,
             maxWriteTimeStampFilter = Long.parseLong(maxWriteTimeStampFilterStr);
         }
 
+        String customWriteTimeStr =
+                sparkConf.get("spark.destination.custom.writeTime", "0");
+        if (null != customWriteTimeStr && customWriteTimeStr.trim().length() > 1) {
```

Comment:
#nitpick shouldn't this be `... && customWriteTimeStr.trim().length() >= 1 && StringUtils.isNumeric(customWriteTimeStr.trim())`?

Reply:
Good catch!